docs/run-taco.md

```diff
@@ -0,0 +1,36 @@
+---
+title: How to Run your Packaged Connector (Taco)
+---
+
+The ability to load taco files was added in 2019.4 Beta 1. For loading un-packaged connectors in earlier versions of Tableau, refer to [How to Run Your "Under Development" Connector]({{ site.baseurl }}/docs/share)
```

Comment:
How about this:
Starting in 2019.4 Beta 1, you can load `.taco` files. To use a connector in earlier Tableau versions, see [Run Your "Under Development" Connector...